examples/extract_xml_field_formats.py

```diff
@@ -155,44 +157,12 @@ def show_xml_field_formats(xml_formats, reference_instrument_name='NIRISS', out_
                                            format='ascii.basic', delimiter=',', overwrite=True)
 
 
-xml_formats = {}
-for instrument in ['NIRISS', 'FGS', 'NIRSpec']:
-
-    siaf = pysiaf.Siaf(instrument)
+xml_formats = OrderedDict()
```

Comment:
I’m not sure exactly how to want to run this file, but this line through the end of the file may be good to put under `if __name__ == "__main__”:`

Reply:
Right. Thanks.